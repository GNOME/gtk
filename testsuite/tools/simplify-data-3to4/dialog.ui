<object class="GtkDialog">
  <child internal-child="vbox">
    <object class="GtkBox">
      <property name="name">fancy</property>
      <child internal-child="action_area">
        <object class="GtkButtonBox">
          <property name="name">actions</property>
        </object>
      </child>
    </object>
    <packing>
      <property name="pack-type">end</property>
    </packing>
  </child>
</object>
