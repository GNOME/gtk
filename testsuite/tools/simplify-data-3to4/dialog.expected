<?xml version="1.0" encoding="UTF-8"?>
<object class="GtkDialog">
  <child internal-child="content_area">
    <object class="GtkBox">
      <property name="visible">0</property>
      <property name="name">fancy</property>
    </object>
    <packing>
      <property name="pack-type">end</property>
    </packing>
  </child>
  <child internal-child="action_area">
    <object class="GtkButtonBox">
      <property name="name">actions</property>
    </object>
  </child>
</object>
